beacon-chain/rpc/subscriber/api/events/backend_service_mock.go

```diff
@@ -0,0 +1,38 @@
+package events
+
+import (
+	"github.com/ethereum/go-ethereum/event"
+	types "github.com/prysmaticlabs/eth2-types"
+	"time"
+)
+
+var (
+	deadline = 5 * time.Minute
+)
+
+type MockBackend struct {
```

Comment:
Here you implement `Backend` an interface but these methods are not defined in that `interface`.

Reply:
Solved. We removed this mock and using another test file - `api_backend_test.go`